interface/src/app/mlmodel/mLModelComponents.js

```diff
@@ -0,0 +1,25 @@
+angular.module('adage.mlmodel.components', [
+  'adage.mlmodel.resource'
+])
+
+.component('mLModelSelector', {
+  templateUrl: 'mlmodel/selector.tpl.html',
+  bindings: {
+    selectedMLModel: '='
+  },
+  controller: ['MLModel', '$log', function(MLModel, $log) {
+    var self = this;
+
+    MLModel.get(
+      function success(response) {
+        self.availableMLModels = response.objects;
+      },
+      function error(err) {
+        $log.error('Failed to get MLModels from REST API: ' + err.statusText);
+        self.errorMessage = 'Failed to get Machine Learning Models from server';
```

Comment:
I like this: simple error message with details logged to the console for follow-up. Although this simple API interaction is not likely to produce too many problems, it's *always* helpful to leave ourselves tools for following up real-world errors.